src/main/java/org/cyclonedx/FileFormat.java

```diff
@@ -0,0 +1,16 @@
+package org.cyclonedx;
+
+public enum FileFormat {
```

Comment:
Nit: Should this be named `Format` instead, since BOMs are not necessarily written to file?

Also, consider adding the [media type](https://github.com/CycloneDX/cyclonedx-core-java/blob/master/src/main/java/org/cyclonedx/CycloneDxMediaType.java) as additional field: https://cyclonedx.org/specification/overview/#registered-media-types

Reply:
> Nit: Should this be named Format instead, since BOMs are not necessarily written to file?

Done.

> Also, consider adding the [media type](https://github.com/CycloneDX/cyclonedx-core-java/blob/master/src/main/java/org/cyclonedx/CycloneDxMediaType.java) as additional field

Done in a separate commit.